packages/core/src/hooks/useLocalStorage.ts

```diff
@@ -1,11 +1,12 @@
 import { useEffect, useState } from 'react'
+import { useConfig } from './useConfig'
 
-function getItem(key: string) {
+function getItem(key: string, storage: WindowLocalStorage['localStorage']) {
   if (typeof window === 'undefined') {
```

Comment:
Not gonna work even with override, because of this.

Add a red test for useLocalStorage with override.